lib/render/editor/editor.html

```diff
@@ -497,9 +508,46 @@
 
       _onExecutionStop: function (data) {
         this._cleanState();
+        this._setResultsBoards(data.test_results);
         mumuki.kids.showResult(data);
       },
 
+      _setResultsBoards: function (testResults) {
+        if (!testResults) return;
+        let testResultsBoards = this._testResultsBoards(testResults);
```

Comment:
We can declare these and the other variables of this file with `const` instead of `let`